magicbyirineu/Modules/CardDetail/View/CardDetailScreen.swift

```diff
@@ -22,16 +22,16 @@ class CardDetailScreen: UIView {
         var view = UIImageView()
         view.contentMode = .scaleAspectFit
         view.clipsToBounds = true
-        view.layer.cornerRadius = 11
+        view.layer.cornerRadius = 20
         return view
     }()
     
     let cardName: UILabel = {
        var view = UILabel()
-       view.font = UIFont(name:"SFProDisplay-Bold", size: 18.0)
+       view.font = UIFont(name:"SFProDisplay-Bold", size: 26.0)
```

Comment:
Uma idéia para a font. Ao invés de ficar escrevendo a Font toda hora, que tal usar algo do tipo `R.swift`?

Outra ideia seria centralizar tudo em uma única class.

Aqui talvez seria interessante, por exemplo, ter uma extension da UIFont onde eu pudesse reutilizar sem precisar ficar escrevendo toda a hora a font pq pode ser perigoso para gerar crashes. 

```suggestion
       view.font = UIFont.sfProDisplay(size: 18.0, type: .bold)
```
Não fica bom?